media-tv/plex-media-server/plex-media-server-1.13.9.5456.ebuild

```diff
@@ -0,0 +1,135 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit user systemd unpacker python-single-r1
+
+DESCRIPTION="A free media library that is intended for use with a plex client."
+HOMEPAGE="http://www.plex.tv/"
+
+COMMIT="ecd600442"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_FULL_VERSION="${PV}-${COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server"
+SRC_URI="
+	amd64? ( ${URI}/${_FULL_VERSION}/plexmediaserver_${_FULL_VERSION}_amd64.deb )
+	x86? ( ${URI}/${_FULL_VERSION}/plexmediaserver_${_FULL_VERSION}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="avahi"
+
+DEPEND="sys-apps/fix-gnustack"
+RDEPEND="avahi? ( "net-dns/avahi" )"
```

Comment:
I see you inherit python-single-r1, but `${PYTHON_DEPS}` is missing. Now, I am not 100 % if it's needed or not, but you should figure it out by reading the python-single-r1 doc. I have a strong feeling that it is required ;)

Reply:
It was added for what I thought was a needed change for running post processing scripts. I was wrong, we didn't need python virtualenv. Removed.

Reply:
`"net-dns/avahi"` these quotation marks are unnecessary and potentially breaking.
`RDEPEND="avahi? ( net-dns/avahi )"` is correct.

The RDEPEND list seems awfully small too, but I'm not saying more until I test it myself.